From 5d99d100f623ea89db7c2dfe6cf1a61d1363a4f4 Mon Sep 17 00:00:00 2001
From: Will Thompson <wjt@endlessos.org>
Date: Fri, 29 Apr 2022 12:58:58 +0100
Subject: [PATCH 10/13] metainfo: Add developer name

Looking at the commit history, it appears that while Armin Burgmeier is
no longer an active contributor, he is responsible for the overwhelming
majority of lines of code changed, and his is the sole name in the
README.

According to the Appstream spec
<https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer_name>:

> This tag is translatable.

I have followed this recommendation on the basis that his name might
reasonably be transliterated into non-Latin writing systems.
---
 gobby-0.5.metainfo.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gobby-0.5.metainfo.xml.in b/gobby-0.5.metainfo.xml.in
index 03ae16b..852836c 100644
--- a/gobby-0.5.metainfo.xml.in
+++ b/gobby-0.5.metainfo.xml.in
@@ -9,6 +9,7 @@
   </provides>
   <_name>Gobby</_name>
   <_summary>Edit text files collaboratively</_summary>
+  <_developer_name>Armin Burgmeier</_developer_name>
 
   <description>
     <_p>
-- 
2.30.2

